.github/workflows/publish_package.yaml

```
@@ -48,10 +48,11 @@ jobs:
       - name: Build the wheel
         run: python setup.py bdist_wheel
 
+      # include beta (main) tags as prod tags, to push to "real" PyPI
```

nit: This comment should document the whole job, not just the change you're making
```suggestion
      # If the tag matches either a release tag or a beta tag, allow publishing to PyPi, eg.
      #   2.10.0 --> match (production release tag)
      #   2.10.0b1 --> match (beta release tag)
      #   2.10.0.dev0 --> no match (development tag)
```

---

adjusted the comment and rebased into one commit to keep a cleaner git history!